task/source-build/0.2/source-build.yaml

```diff
@@ -182,7 +182,7 @@ spec:
           --source-dir "$SOURCE_DIR"
           --base-images "$base_images"
           --write-result-to "$RESULT_FILE"
-          --cachi2-artifacts-dir "$CACHI2_ARTIFACTS_DIR"
+          --prefetch-artifacts-dir "$CACHI2_ARTIFACTS_DIR"
```

Comment:
0.2 will be deprecated. No need to update here.

Reply:
> ```yaml
> build.appstudio.redhat.com/expires-on: "2025-08-12T00:00:00Z"
> ```

https://github.com/konflux-ci/build-definitions/blob/main/task/source-build/0.2/source-build.yaml#L9